### PR TITLE
[doc] schema metrics doc

### PR DESCRIPTION
**Pull request by tjiuming**
### Motivation

add doc for https://github.com/apache/pulsar/pull/14723

### Modifications

add doc in reference-metrics.md

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by tjiuming** [comment]
飞书20220327-235556
飞书20220327-235602

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+
+> For a functions-worker running separately from brokers, its schema metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
+
+All the schema metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of delete schema failed operations. |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:509]:
```suggestion
| pulsar_schema_del_ops_failed_count | Counter | Number of operations that delete schemas |
```
do you intend to mean this？
if so, could you please help update other occurencies? thanks

**Review by gaoran10 — APPROVED** [review]

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+
+> For a functions-worker running separately from brokers, its schema metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
+
+All the schema metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of operations that delete schemas failed. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:509]:
```suggestion
| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
```

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+| pulsar_schema_get_ops_failed_count | Counter | Number of operations that get schemas failed. |
+| pulsar_schema_put_ops_failed_count | Counter | Number of operations that put schemas failed. |
+| pulsar_schema_compatible_count | Counter | Number of schema compatible operations. |
+| pulsar_schema_incompatible_count | Counter | Number of schema incompatible operations. |
+| pulsar_schema_del_ops_latency | Summary | Latency of operations which delete schema succeed. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:514]:
```suggestion
| pulsar_schema_del_ops_latency | Summary | Latency of successful operations to delete schemas. |
```

This sentence pattern applies to others.

File: site2/docs/reference-metrics.md [hunk 4]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
+| pulsar_schema_get_ops_failed_count | Counter | Number of failed operations to get schemas. |
+| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to put schemas. |
+| pulsar_schema_compatible_count | Counter | Number of schema compatible. |
+| pulsar_schema_incompatible_count | Counter | Number of schema incompatible. |
+| pulsar_schema_del_ops_latency | Summary | Latency of successful operations to delete schemas. |
+| pulsar_schema_get_ops_latency | Summary | Latency of successful operations to get schema. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:515]:
```suggestion
| pulsar_schema_get_ops_latency | Summary | Latency of successful operations to get schemas. |
```

File: site2/docs/reference-metrics.md [hunk 5]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+| pulsar_schema_put_ops_latency | Summary | Latency of successful operations to put schema. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:516]:
```suggestion
| pulsar_schema_put_ops_latency | Summary | Latency of successful operations to send schemas. |
```

File: site2/docs/reference-metrics.md [hunk 6]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
+| pulsar_schema_get_ops_failed_count | Counter | Number of failed operations to get schemas. |
+| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to put schemas. |
+| pulsar_schema_compatible_count | Counter | Number of schema compatible. |
+| pulsar_schema_incompatible_count | Counter | Number of schema incompatible. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:513]:
```suggestion
| pulsar_schema_incompatible_count | Counter | Number of incompatible schemas. |
```

File: site2/docs/reference-metrics.md [hunk 7]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+All the schema metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
+| pulsar_schema_get_ops_failed_count | Counter | Number of failed operations to get schemas. |
+| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to put schemas. |
+| pulsar_schema_compatible_count | Counter | Number of schema compatible. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:512]:
```suggestion
| pulsar_schema_compatible_count | Counter | Number of compatible schemas. |
```

File: site2/docs/reference-metrics.md [hunk 8]
```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
...
+
+All the schema metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of failed operations to delete schemas. |
+| pulsar_schema_get_ops_failed_count | Counter | Number of failed operations to get schemas. |
+| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to put schemas. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:511]:
```suggestion
| pulsar_schema_put_ops_failed_count | Counter | Number of failed operations to send schemas. |
```

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**